src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -728,7 +696,7 @@ public String getValidityInfoForNonNullField(String fieldName, Object value) {
         return value == null ? NON_NULL_FIELD_ERROR_MESSAGE.replace("${fieldName}", fieldName) : "";
     }
 
-    private boolean isUntrimmed(String value) {
+    public static boolean isUntrimmed(String value) {
```

Comment:
@whipermr5  Now using Inheritance for FieldValidator  but i don't think there is need of inheritance  in other classes.
Also if using inheritance i.e` teammates.test.driver.StringHelper  extends teammates.common.util.StringHelper` then there are other problems like making util.StringHelper as abstract class.
Please see the new changes